.circleci/config.yml

```diff
@@ -151,6 +156,13 @@ jobs:
             popd
             ghr -t "${GITHUB_TOKEN}" -u "${CIRCLE_PROJECT_USERNAME}" -r "${CIRCLE_PROJECT_REPONAME}" -c "${CIRCLE_SHA1}" -delete  "${CIRCLE_TAG}" ./pyodide-build-${CIRCLE_TAG}.tar.bz2
 
+      - run:
+          name: Deploy to pyodide-cdn2.iodide.io
+          command: |
+            aws s3 sync --delete build/ "s3://pyodide-cdn2.iodide.io/v${CIRCLE_TAG}/full/" --cache-control max-age=30758400  # 1 year
+            # update 301 redirect for the /latest/* route.
+            aws s3api put-bucket-website --cli-input-json file://.circleci/s3-website-config.json
+
```

Comment:
I wonder if we should take out the netlify deploy? At this point I'm worried that it will just cause confusion.

I'd like to work on grandfathering it out on some timeframe (maybe 6 months or something)